cmake/FDBInstall.cmake

```diff
@@ -0,0 +1,258 @@
+function(fdb_install_packages)
+  set(FDB_INSTALL_PACKAGES ${ARGV} PARENT_SCOPE)
+endfunction()
+
+function(fdb_install_dirs)
+  set(FDB_INSTALL_DIRS ${ARGV} PARENT_SCOPE)
+endfunction()
+
+function(install_symlink_impl)
+  if (NOT WIN32)
```

Comment:
nitpick: an early return could save indentation

Reply:
I think this is a question of taste (`return` tends to have similar problems like `goto` - I guess you can just see who my programming teacher in college was ;) ).

But I agree the indent might be a bit distracting - so I will rewrite this.